nerdlandbot/__main__.py

```diff
@@ -47,6 +47,7 @@
 bot.load_extension("nerdlandbot.commands.poll")
 bot.load_extension("nerdlandbot.commands.purger")
 bot.load_extension("nerdlandbot.commands.kerk")
+bot.load_extension("nerdlandbot.commands.dadJoke")
```

Comment:
To keep it consistent, could you change it to snake case (spaces become _'s) like in random_user? It's also python convention to use snake case rather than camelcase